Change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
<p>you can give meaningful title in 1st markdown cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>